FileCopy/utils.h

```diff
@@ -1,6 +1,11 @@
 #ifndef FILECOPY_UTILS
 #define FILECOPY_UTILS
 
-void setUpDebugLogging(const char *logname, int argc, char *argv[]);
+void setup_logging(const char *logname, int argc, char *argv[]);
+bool check_directory(const char *dirname);
```

Comment:
This can return `void`, and maybe call it `is_directory` to match the `is_file` function?